infrastructure/httpapi/handlers/validators.go

```diff
@@ -298,6 +303,55 @@ func (handler *Validators) getAverageBlockTime() (*big.Float, error) {
 		big.NewFloat(1000),
 	)
 
+	// In ideal case, we calculate the average block time with recent N most blocks (blocks in M recent days).
+	// N = M * 24 * 3600 / averageBlockTime
+
+	nRecentBlocksInFloat := new(big.Float).Quo(
```

Comment:
should N be static?

Reply:
Thanks for the advice! Now it is changed to be static~

Reply:
Thanks!

Reply:
And i guess we can remove the existing averageBlockTime calculation part as it seems no longer needed anymore, the part that using total block time.